### ISSUE TITLE
Using MTP access breaks the flash

### ISSUE THREAD

**Issue by ghost** [DirtyUnicorns/android_device_oneplus_bacon#1]
I've installed DU on my OPO yeasterday and each time I've used MTP since then, I've had to flash the ROM again after. It happens when I unplug the USB. The phone freezes, restart and then freezes agin during the load animation. After that, it loops like that unless I restart on Clockwork Mod and flash the ROM again.

**Comment by spaceman860** [comment]
This is NOT the place to report your issues.

**Comment by AugierLe42e** [comment]
Ok... Well do what you want then... At least you are aware...

**Comment by jmztaylor** [comment]
Http://jira.dirtyunicorns.com

**Comment by AugierLe42e** [comment]
@jmztaylor: Ok, I'll report. But, if I may, using a closed Jira is not the most conveniant. Having to subscribe to report a bug is not the best way to get feedbacks ;)

**Comment by nychitman1** [comment]
This is how we take bug reports. If you don't like it - do not submit anything. That is all there is to say on the matter.

**Comment by AugierLe42e** [comment]
Interresting... Free devs asking for feedback and being agressive when getting it...

**Comment by nychitman1** [comment]
It's not aggression. The feeling you see is frustration because if you took the time to read the OP of any of our threads - we specifically state that we take bug reports on Jira. Nowhere does it state go and post bug reports on Github.

**Comment by AugierLe42e** [comment]
Look, I've just tried to install a ROM for my phone. It's not my fault if your organisation is a bit confusing. You have a [website](http://dirtyunicorns.com/dusite/) which seem to be useless, a [G+ account](https://plus.google.com/communities/109738128866939227235) which is not clear if you are using it as a website and, finally, I had to grad the ROM on [XDA forums](http://forum.xda-developers.com/oneplus-one/orig-development/rom-dirty-unicorns-bacon-t3077755). You store the source on Github, but using the Jira, I discovered you also have a [Gerrit](http://gerrit.dirtyunicorns.com/#/q/status:open,n,z).

Believe it or not, but I'm just like you: I have more usefull things to do than trying to understand how your work with all your things. I'm a goodwill user and I just tried to help you. So, sincerely, let me apologize and wish you good luck for the future.
